components/about_build_modal/about_build_modal.tsx

```diff
@@ -278,7 +278,7 @@ export default class AboutBuildModal extends React.PureComponent<Props, State> {
                         <p>
                             <FormattedMarkdownMessage
                                 id='about.notice'
-                                defaultMessage='Mattermost is made possible by the open source software used in our [server](!https://about.mattermost.com/platform-notice-txt/), [desktop](!https://about.mattermost.com/desktop-notice-txt/) and [mobile](!https://about.mattermost.com/mobile-notice-txt/) apps.'
+                                defaultMessage='Mattermost is made possible by the open source software used in our [server](!https://github.com/mattermost/mattermost-server/blob/master/NOTICE.txt), [desktop](!https://github.com/mattermost/desktop/blob/master/NOTICE.txt) and [mobile](!https://github.com/mattermost/mattermost-mobile/blob/master/NOTICE.txt) apps.'
```

Comment:
Note that these links may break if we ever move around the code in those repos. I think the reason that the old version pointed to about.mattermost.com was that we had redirects set up so that older versions of the app  would still link to the correct place in case the repos ever changed

Reply:
Good point, @hmhealey. There will be redirects in place via Marketing following the switchover; however, ideally we don't want to reply on those alone. I'm 2/5 for applying these URL updates as well. Given the number of instances of these strings alone in the product, we should align on an approach.

@emdecr - What is your recommendation for situations like where we've got in-product links pointing to legacy domains intentionally to avoid future rework? Where would you say we draw the line between updated domain links and keeping existing legacy links in place?

Reply:
@cwarnermm Hmm good to know. I'm leaning towards still replacing them since we're phasing out this subdomain entirely. We have the ability to update redirects, which is what would have to happen with this old setup (ie. if `about.mm.com/platform-notice-text/` needed to be pointed at something else, we'd have to update the redirect anyways).

What might help is to map out which URLs are considered as these types of stand-in links, and make mm.com equivalents. 

Does that make sense/help at all? Am I understanding the issue, or am I way off base?

Reply:
That sounds like it would help for if the linked pages ever change in the future (like if the NOTICE.txt in the GitHub repo moves or AWS moves a docs page that we're linking to), but it doesn't fix the issue that older instances will still be pointing at `about.mattermost.com` URLs so those links will break. That will however be fixed over time as people update to newer versions of MM.

The bigger problem I see is for the configurable links stored in the config.json since even if we change the defaults, they'll never be updated on their own, so we could end up with someone's config still pointing to `about.mattermost.com` even if they do update to a newer version of MM. We could go and make it so that those links are rewritten by the server code on upgrade though.

Reply:
@hmhealey - This is ultimately a conversation and decision to be made jointly between Marketing and Engineering that balances potential risk against future-proofing. One key thing is that while these domains are going away for good, redirects _will_ be in place, so we have some time to make an appropriate future-proofed decision that doesn't have to be now.

What we do need to decide now, as part of the domain consolidation effort, is whether we keep these URL updates as-is or revert them in this repo.
- if we leave them as-is, and Engineering refactors code in the future, those links may break. That feels like the cost of a refactor in general.
- if we choose to revert them, I agree with @emdecr that we'll need to account for this in the future somehow since these domains are being sunsetted permanently. 

With this in mind, how do you both want to proceed, @hmhealey and @emdecr?

Reply:
Thanks for the all the context @hmhealey.

Just to be safe, I'm fine with making the call to **revert back to keeping these about.mm.com links in-tact until we have more eyes/ears on this topic**. 

This probably merits a larger conversation once the dust settles re: replacing these URLs, and a strategy moving forward. Might be worth it to make a product-specific subdomain vs referencing any type of user-facing site. Then we can use that subdomain as the redirect hub for all these stand-in links.

We're not going to ax the about.mm.com redirects anytime soon either. So, people will end up at destinations vs being served a 404.